### ISSUE TITLE
Add Cloud Workflow resource support

### ISSUE THREAD

**Issue by tokatsu-gc** [GoogleCloudPlatform/k8s-config-connector#511]
**Describe the feature or resource**
Please add Google Cloud Workflow as a supported resource.
A similar feature is also supported by terraform.
https://cloud.google.com/workflows/docs/quickstart-terraform

**Importance**
The number of Serverless entities are increasing and workflow of serverless functions are getting more complicated.
Managing workflow configuration with Config Connector is a preferable way to manage serverless workflow on Google Cloud.

**Comment by jcanseco** [comment]
Thank you @tokatsu-gc.

> A similar feature is also supported by terraform.

To clarify, you're referring to the [google_workflows_workflow](https://registry.terraform.io/providers/hashicorp/google/latest/docs/resources/workflows_workflow) resource right?

If so, I've added the resource request to our backlog.

**Comment by tokatsu-gc** [comment]
Yes, I would like Config Connector to support the [google_workflows_workflow](https://registry.terraform.io/providers/hashicorp/google/latest/docs/resources/workflows_workflow) resource.

**Comment by jcanseco** [comment]
Great, thanks for confirming!

**Comment by tokatsu-gc** [comment]
When is the ETA of supporting [google_workflows_workflow](https://registry.terraform.io/providers/hashicorp/google/latest/docs/resources/workflows_workflow)?

**Comment by mbzomowski** [comment]
At the moment, it looks like it should be added at 2022Q2 at the earliest.

**Comment by diviner524** [comment]
@vgelot Sorry we don't have a solid ETA for cloud workflow resource at this moment. Also if this is a blocker for you to adopt Config Connector, please consider filing a [GCP support](https://cloud.google.com/support-hub) ticket to boost the signal and help the team better prioritize our resource requests.

**Comment by davidvarnes** [comment]
Connector support for Workflows is important to our adoption of Config Connector in the serverless landscape

**Comment by wtait1-ff** [comment]
I just created a [request](https://issuetracker.google.com/issues/303699858) in IssueTracker to mirror this Github issue.

**Comment by diviner524** [comment]
We have a preview CRD (v1alpha1) to support this resource.

https://github.com/GoogleCloudPlatform/k8s-config-connector/blob/master/crds/workflows_v1alpha1_workflowsworkflow.yaml

Here is link to instructions on how to install and manage the v1alpha1 CRDs:

https://cloud.google.com/config-connector/docs/how-to/install-alpha-crds

**Comment by diggerdric** [comment]
Any timeline on this going into beta? 🤞 
Looks like it's really close...
https://github.com/GoogleCloudPlatform/k8s-config-connector/pull/4469

**Comment by ldanielmadariaga** [comment]
This item has been open for >12 months. As part of scheduled repository maintenance, this Issue/PR will be closed. If this is still a critical ask, please re-open and work with your Google Cloud Account team to prioritize it with GCP KCC Product Management.
